js.go

```diff
@@ -118,6 +118,9 @@ type JetStream interface {
 	// ChanSubscribe creates channel based Subscription.
 	ChanSubscribe(subj string, ch chan *Msg, opts ...SubOpt) (*Subscription, error)
 
+	// QueueChanSubscribe creates channel based Subscription with a queue group.
```

Comment:
Does not match the API name `ChanQueueSubscribe()`

Reply:
Fixed now.